extensions/_beacons/http_status.py

```diff
@@ -81,28 +97,30 @@ def beacon(config):
             - sites:
                 example-site-1:
                   url: "https://example.com/status"
-                  json_response:
-                    - path: 'redis:status'
-                      value: 'up'
-                      comp: '='
-                    - path: 'postgresql:response_microseconds'
-                      value: 50
-                      comp: '<='
-                  html_response:
-                    - path: ''
-                      value: 'foo.*bar'
-                      comp: search
+                  status:
```

Comment:
Not critical for this, but might be useful to add all of the options to the pillar example (e.g. timeout, content_type)

Reply:
Added